Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt

```diff
@@ -163,14 +164,20 @@ private fun MapViewContainer(
         // Reading zoom so that AndroidView recomposes when it changes. The getMapAsync lambda
         // is stored for later, Compose doesn't recognize state reads
         val mapZoom = zoom
+        if (!mapInitialized) {
```

Comment:
Probably better done in a launched effect?

```
LaunchedEffect(mapInitialized) {
    if (!mapInitialized) {
        // blah
    }
}
```

Reply:
Yeah! Much better, thanks!